safekeeper/src/timeline.rs

```diff
@@ -87,6 +88,7 @@ struct SharedState {
     active: bool,
     num_computes: u32,
     pageserver_connstr: Option<String>,
+    last_removed_segno: XLogSegNo,
```

Comment:
Not directly related to this PR, but I'd like also to have `disk_start_lsn` stored in the control file, for the minimal LSN available locally.